mmf/configs/defaults.yaml

```diff
@@ -33,8 +33,8 @@ training:
     # Whether MMF should log or not, Default: False, which means
     # mmf will log by default
     should_not_log: false
-    # Whether to capture stdout logs by the logger. Default: True
-    stdout_capture: true
```

Comment:
Are we not capturing stdout logs anymore or not having an option to do so? Or did I miss something?

Reply:
We are capturing stdout always. There is a should_not_log option that basically does the same thing as setting this to false.

Reply:
`stdout_capture` here means capturing stdout to log files. It is different from `should_not_log`.

Reply:
Yes, discussed offline. The option isn't useful anymore.